psydac/api/ast/parser.py

```diff
@@ -1233,7 +1233,7 @@ def _visit_ComputeKernelExpr(self, expr, op=None, lhs=None, **kwargs):
         lhs = lhs[:]
 
         # Create a new name for the temporaries used in each patch
-        name=lhs[0]._name[12:-8]
+        name = get_name(lhs)
```

Comment:
If you execute the code I wrote in the issue, you'll have 
`lhs = [0, contribution_w_1_u_1_09m4547t]`
Trying to access the `_name` attribute of 0 would cause the bug to appear.

Reply:
That's interesting. Thanks!